update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_DIRECTORY="https://archive.raspbian.org"
+RASPBIAN_ARCHIVE_KEY_FILE_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_DIRECTORY}/${RASPBIAN_ARCHIVE_KEY_FILE_NAME}"
```

Comment:
Because of the [DRY principle](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself)